state/scale.go

```diff
@@ -0,0 +1,60 @@
+package state
+
+import (
+	"github.com/hashicorp/nomad-autoscaler/state/status"
+	"github.com/hashicorp/nomad/api"
+)
+
+// statusUpdate is responsible for taking a Nomad JobScaleStatusResponse and
+// updating the autoscaler internal state.
+func (h *Handler) statusUpdate(update *api.JobScaleStatusResponse) {
+
+	// If the job is stopped, remove the scaling policies, remove the scale
+	// status handler and delete the scale status state. Even if the job is
+	// started again, the Autoscaler will gather all the information it
+	// needs so we can clear everything out.
+	switch update.JobStopped {
```

Comment:
Is `JobStopped` just a `boolean`? If so, I think an `if` statement would be better here.

Reply:
Updated locally.